overlord/devicestate/handlers.go

```diff
@@ -930,6 +930,20 @@ func makeRollbackDir(name string) (string, error) {
 	return rollbackDir, nil
 }
 
+var (
```

Comment:
I'm a bit worried that handlers.go and devicestate_test.go here are growing. It might be time to split things:

* handlers_serial.go  "generate-device-key" "request-serial" and helpers
* handlers_gadget.go   the gadget stuff
* handler.go the rest (for now)

and split/rename handlers_test.go and devicestate_test.go as needed.

Reply:
And also see if some of the unexported functions should go to helpers.go. They probably should if they end up used by two other files.